### PR TITLE
feat: add dotnet sample app

### PR DESCRIPTION
**Pull request by pankajagrawal16**
Related issue: https://github.com/Azure/aca-dotnet-workshop/issues/77

- Deploy dotnet sample app used in workshop https://azure.github.io/aca-dotnet-workshop/
- Eventually this will be referenced in the workshop itself to have a guided experience of how to deploy the app in landing zone for aca as well.

### CODE REVIEW

File: scenarios/shared/bicep/private-networking.bicep [hunk 1]
```diff
@@ -36,15 +36,15 @@ param location string = resourceGroup().location
...
+  name: 'privateDnsZoneDeployment-${uniqueString(azServiceId, privateEndpointSubResourceName)}'
   params: {
     name: azServicePrivateDnsZoneName
     virtualNetworkLinks: virtualNetworkLinks
   }
 }
 
 module privateEndpoint 'private-endpoint.bicep' = {
-  name: 'privateEndpointDeployment-${uniqueString(azServiceId)}'
+  name: 'privateEndpointDeployment-${uniqueString(azServiceId, privateEndpointSubResourceName)}'
```
Comment (pankajagrawal16) [inline scenarios/shared/bicep/private-networking.bicep:47]: This is need for when you have multiple sub resource name  to create private endpoint on. For example storage account with queue and blob. Else deployment replaces once created after

File: scenarios/shared/bicep/naming/naming.module.bicep [hunk 2]
```diff
@@ -61,6 +61,8 @@ var resourceNames = {
   logAnalyticsWorkspace: replace(namingBase, resourceTypeToken, naming.resourceTypeAbbreviations.logAnalyticsWorkspace)
   serviceBus: replace(namingBaseUnique, resourceTypeToken, naming.resourceTypeAbbreviations.serviceBus)
   serviceBusPep: '${naming.resourceTypeAbbreviations.privateEndpoint}-${replace(namingBaseUnique, resourceTypeToken, naming.resourceTypeAbbreviations.serviceBus)}'
+  storageAccount: toLower(take( replace ( replace(namingBaseUnique, resourceTypeToken, naming.resourceTypeAbbreviations.storageAccount), '-', ''), 24))
```
Comment (pankajagrawal16) [inline scenarios/shared/bicep/naming/naming.module.bicep:64]: Support for storage account naming.

**Review by thotheod — APPROVED** [review]
Thanks for the submission @pankajagrawal16 . I 've done some minor fixes  (mainly in docs, or coding standards formatting). If you have any questions ping me

**Comment by pankajagrawal16** [comment]
Great. Thanks @thotheod . Looks good to me.

**Comment by aarthiem** [comment]
Looks good, Thanks for your contribution!

**Review by aarthiem — APPROVED** [review]

**Pull request merged**